tests/test_user_model.py

```diff
@@ -141,13 +141,14 @@ def test_anonymous(self):
     def test_agency_worker_role(self):
         Role.insert_roles()
         r = Role.query.filter_by(permissions=Permission.AGENCY_WORKER).first()
-        a = Agency(name='SEPTA')
+        ag_1 = Agency(name='SEPTA')
+        ag_2 = Agency(name='STREETS')
```

Comment:
I think this is also fine. We are creating an agency with an arbitrary name. It doesn't really matter what name we give it (just like it's fine for us to create a test user named 'Joe').